scripts/email/helpers/firebase_users_loader.py

```diff
@@ -0,0 +1,76 @@
+import firebase_admin
+from firebase_admin import credentials, firestore
+from collections import defaultdict
+from typing import Dict, List, Tuple
+
+cred = credentials.Certificate("./serviceAccountKey.json")
+firebase_admin.initialize_app(cred)
+
+db = firestore.client()
+
+
+def get_all_user_names() -> Dict[str, Tuple[str, str, str]]:
+    print("Fetching all user names from Firebase...", end="\n\n")
+    user_names = {}
+    docs = db.collection("user-name").get()
+    for doc in docs:
```

Comment:
is it necessary to extract specific user information if we are just using "Hey, courseplanner" in the email header? Could be worth it to keep in the future if we decide to add personalization, but we can possibly omit this for now if it isn't needed

Reply:
Good point. I'll comment this out for now.